docs/cookbook/pelias/index.rst

```diff
@@ -0,0 +1,49 @@
+Pelias Search (Forward geocoding)
+=================================
+
+Problem
+-------
+
+You'd like to use the `Pelias geocoder <https://github.com/pelias/pelias>`_ (or an equivalent online service, like `geocode.earth <https://geocode.earth/>`_).
+
+This was previously also known as Mapzen Search, although those services are end-of-life.
+
+
+Solution
+--------
+
+Add suitable config settings for the service you want to enable. 
+
+The base application includes the Pelias plugin, but it is disabled until configured. The following code shows how to change the config settings for Pelias from geocode.earth, where your API key goes in place of the REPLACE_THIS part of the :code:`url`:
+
+.. code-block:: json
+  :caption: ``config/settings.json``
+
+  {
+    "admin": {
+      "plugin": {
+        "pelias": {
+          "geocoder": {
+            "url": "https://api.geocode.earth/v1/search?api_key=REPLACE_THIS&text={s}",
```

Comment:
While this will work perfectly fine, we always recommend hiding your API key behind a proxy that you control so that it is not visible to end users. That way it is less likely to be abused in some fashion.

Reply:
Is this something you were planning for the deployment guide (#99)? Might it be worth a separate cookbook example?

I can see a simple pass-through handler on the server side (e.g. Apache with mod_proxy / mod_rewrite), but is this something that `os.net.ProxyHandler` can make transparent?

Reply:
That is a job for a simple proxypass server-side.

The purpose of `ProxyHandler` is to pass failed requests through a generic proxy to avoid mixed content and CORS issues. URLs going through it are still created client side and the user can see everything in the request payload if they are so inclined.

Reply:
I think I've addressed the reverse proxy needs (which took way longer than it should have) in https://github.com/ngageoint/opensphere/pull/95/commits/fa373208cf5e6b0161a683a5c40881cd54f28a15.